safe-global/safe-wallet-web
# chore: update terms and conditions request [SW-57]

## What it solves
We have to ask all our users to reconfirm the cookies and terms and conditions. 

## How to test it
Open the page you should immediately see the cookie and terms prompt. There is no checkbox for terms, but it is implied that by using the page they are accepted

## Screenshots

## Checklist
